src/client/webpack.config.js

```diff
@@ -43,7 +42,12 @@ const webpackConfig = {
     ]),
     new webpack.optimize.CommonsChunkPlugin(/* chunkName= */'vendor', /* filename= */'vendor.js', function(module){
       return module.resource && module.resource.indexOf('node_modules') !== -1;
-    })
+    }),
+    new webpack.DefinePlugin({
+     __CONFIG__: args.endpoint
```

Comment:
I quote like the alias approach. 

Makes the usage more explicit. I tend to think of the define plugin in the same context as magic global variables. 

With the alias the scope of the configuration is better defined. It's still somewhat magic, but at least the scope is better defined.